mltsp/util.py

```diff
@@ -146,3 +147,16 @@ def cast_model_params(model_type, model_params):
         if isinstance(model_params[k], str) and str not in dest_types_list:
             raise(ValueError("Model parameter cannot be cast to expected "
                              "type."))
+
+
+def remove_files(paths):
```

Comment:
I'm actually using this function as it is in `datasets` but making a context manager wrapper could be neat...